data/maps/BattleFrontier_BattleDomeBattleRoom/scripts.inc

```diff
@@ -476,6 +476,7 @@ BattleFrontier_BattleDomeBattleRoom_EventScript_SetUpObjects::
 	call BattleFrontier_EventScript_SetBrainObjectGfx
 	setobjectxyperm LOCALID_OPPONENT, 13, 9
 	removeobject LOCALID_OPPONENT
+	delay 1
```

Comment:
I'm unsure what this delay has to do with DNS. If anything, isn't this potentially an issue with dynamic palettes and thus an issue on master too?

Reply:
The commit labeled as "Fixed shadow positioning bug in some battle facilities", which means that it's relevant to this PR and not current master.